### PR TITLE
Changes for Remove icons task

### PR DESCRIPTION
**Pull request by JimBratsos**
What I did:
Removed icons
Added links
Fixed the reference to the icons themselves ( github was linkedin and linkedin was github )

### CODE REVIEW

File: src/components/LandingPage/sections/SectionThree/SectionThreeIcons/SectionThreeIcons.js [hunk 1]
```diff
@@ -25,17 +25,20 @@ const icons = importAll(
...
 const SectionThreeIcons = () => (
     <div className={classes.socialIcons}>
+
         <ParallaxLayer
             offset={2.54}
             speed={0.5}
             id={classes.github}
             style={{ width: 'min-content', height: '0', zIndex: '99' }}
         >
+            <a href = "https://github.com/zero-to-mastery/visual-music">
```
Comment (ArnasDickus) [inline src/components/LandingPage/sections/SectionThree/SectionThreeIcons/SectionThreeIcons.js]: Add `target="_blank"`

File: src/components/LandingPage/sections/SectionThree/SectionThreeIcons/SectionThreeIcons.js [hunk 2]
```diff
@@ -44,26 +47,11 @@ const SectionThreeIcons = () => (
             id={classes.linkedIn}
             style={{ width: 'min-content', height: '0', zIndex: '99' }}
         >
+            <a href = "https://www.linkedin.com/company/zerotomastery/">
```
Comment (ArnasDickus) [inline src/components/LandingPage/sections/SectionThree/SectionThreeIcons/SectionThreeIcons.js]: Add `target="_blank"`

**Review by ArnasDickus — CHANGES_REQUESTED** [review]
Hello, thanks for adding PR. Next time create a PR from your development branch instead of working directly in your development branch. Because:
1) Until this PR is approved, you can't work on any other PR.
2) It shows that 3 participants have worked on this PR. Which is false.

**Review by JimBratsos — COMMENTED** [review]
Changes requested are completed.

**Comment by ArnasDickus** [comment]
@JimBratsos Okay, now the only thing that left is you need to fix conflicting files. By updating your branch. Try running on development branch
`git fetch upstream`
`git merge upstream/development` 
you might encounter some conflicts that you need to fix manually, and commit again.

**Review by ArnasDickus — APPROVED** [review]

**Pull request merged**